ingester2/src/server/grpc/query.rs

```diff
@@ -187,13 +187,13 @@ where
             .await
         {
             Ok(v) => v,
-            Err(e) => {
-                error!(
-                    error=%e,
-                    %namespace_id,
-                    %table_id,
-                    "query error"
-                );
+            Err(e @ QueryError::TableNotFound(_, _) | e @ QueryError::NamespaceNotFound(_)) => {
```

Comment:
You _might_ be able to do:

```suggestion
            Err(e @ (QueryError::TableNotFound(_, _) |  QueryError::NamespaceNotFound(_)) => {
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=e99f8726bba53d6da6f6cd27b904369b